press/press/doctype/site/site.py

```diff
@@ -710,23 +716,30 @@ def space_required_on_db_server(self):
 		db_size = frappe.get_doc("Remote File", self.remote_database_file).size
 		return 8 * db_size * 2  # double extracted size for binlog
 
-	def check_enough_space_on_server(self):
-		app: "Server" = frappe.get_doc("Server", self.server)
-		db: "DatabaseServer" = frappe.get_doc("Database Server", app.database_server)
-		app_server_free_space = app.free_space
-		db_server_free_space = db.free_space
-		if (diff := app_server_free_space - self.space_required_on_app_server) <= 0:
-			frappe.throw(
-				f"Insufficient estimated space on Application server to create site. Required: {human_readable(self.space_required_on_app_server)}, Available: {human_readable(app_server_free_space)} (Need {human_readable(abs(diff))})",
-				InsufficientSpaceOnServer,
-			)
+	def check_and_increase_disk(self, server: "BaseServer", space_required: int):
+		if (diff := server.free_space - space_required) <= 0:
+			msg = f"Insufficient estimated space on Application server to create site. Required: {human_readable(self.space_required_on_app_server)}, Available: {human_readable(server.free_space)} (Need {human_readable(abs(diff))})."
```

Comment:
Here `space_required` and `self.space_required_on_app_server` aren't the same at times.
ref: #21703